mappings/net/minecraft/item/Vanishable.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/class_5150 net/minecraft/item/Vanishable
+	COMMENT Represents a type of item that can have the "curse of vanishing", or the
+	COMMENT {@code minecraft:vanishing_curse}, enchantment.
```

Comment:
Sounds like you said the same thing twice

Reply:
the displayed name isn't the same as the id, so I said the thing twice. [And someone think it is actually unintuitive.](https://minecraft.gamepedia.com/Official_pages/Java_Edition_unintuitive_ID_list#Enchantment_IDs)

Reply:
It's on that list because it's inconsistent, not necessarily unintuitive. No-one finds that name unintuitive. And besides, you don't need to reference something by its ID if it already has a name. On the rare occasion you need an ID you can look it up in the `Enchantments` class.